lib/pure/base64.nim

```diff
@@ -144,41 +144,38 @@ template encodeImpl() {.dirty.} =
 proc encode*[T: SomeInteger|char](s: openArray[T], safe = false): string =
   ## Encodes `s` into base64 representation.
   ##
-  ## This procedure encodes an openarray (array or sequence) of either integers
-  ## or characters.
-  ##
   ## If `safe` is `true` then it will encode using the
   ## URL-Safe and Filesystem-safe standard alphabet characters,
   ## which substitutes `-` instead of `+` and `_` instead of `/`.
   ## * https://en.wikipedia.org/wiki/Base64#URL_applications
   ## * https://tools.ietf.org/html/rfc4648#page-7
   ##
   ## **See also:**
-  ## * `encode proc<#encode,string>`_ for encoding a string
   ## * `decode proc<#decode,string>`_ for decoding a string
   runnableExamples:
+    assert encode("Hello World") == "SGVsbG8gV29ybGQ="
     assert encode(['n', 'i', 'm']) == "bmlt"
     assert encode(@['n', 'i', 'm']) == "bmlt"
     assert encode([1, 2, 3, 4, 5]) == "AQIDBAU="
   encodeImpl()
 
-proc encode*(s: string, safe = false): string =
-  ## Encodes `s` into base64 representation.
-  ##
-  ## This procedure encodes a string.
-  ##
-  ## If `safe` is `true` then it will encode using the
-  ## URL-Safe and Filesystem-safe standard alphabet characters,
-  ## which substitutes `-` instead of `+` and `_` instead of `/`.
-  ## * https://en.wikipedia.org/wiki/Base64#URL_applications
-  ## * https://tools.ietf.org/html/rfc4648#page-7
-  ##
-  ## **See also:**
-  ## * `encode proc<#encode,openArray[T]>`_ for encoding an openarray
-  ## * `decode proc<#decode,string>`_ for decoding a string
-  runnableExamples:
-    assert encode("Hello World") == "SGVsbG8gV29ybGQ="
-  encodeImpl()
+# proc encode*(s: string, safe = false): string =
+#   ## Encodes `s` into base64 representation.
+#   ##
+#   ## This procedure encodes a string.
+#   ##
+#   ## If `safe` is `true` then it will encode using the
+#   ## URL-Safe and Filesystem-safe standard alphabet characters,
+#   ## which substitutes `-` instead of `+` and `_` instead of `/`.
+#   ## * https://en.wikipedia.org/wiki/Base64#URL_applications
+#   ## * https://tools.ietf.org/html/rfc4648#page-7
+#   ##
+#   ## **See also:**
+#   ## * `encode proc<#encode,openArray[T]>`_ for encoding an openarray
+#   ## * `decode proc<#decode,string>`_ for decoding a string
+#   runnableExamples:
+#     assert encode("Hello World") == "SGVsbG8gV29ybGQ="
+#   encodeImpl()
```

Comment:
Do not comment out whole procs, remove it or improve it, but not let it commented out like that IMHO. :thinking:

Reply:
Oh, I thought I removed it.